firmware/src/dmx.c

```diff
@@ -65,7 +65,8 @@ void DMX_Tasks() {
   }
 }
 
-void DMX_BeginFrame(const uint8_t* data, unsigned int size) {
+void DMX_BeginFrame(uint8_t start_code, const uint8_t* data,
```

Comment:
I'm actually part way through writing the DMX side. Don't worry, I'll use the start code!